src/field_effect.c

```diff
@@ -1300,7 +1301,8 @@ static void CreateHofMonitorSprite(s16 taskId, s16 x, s16 y, bool8 isSmallMonito
     {
         spriteId = CreateSpriteAtEnd(&sSpriteTemplate_HofMonitorBig, x, y, 0);
         SetSubspriteTables(&gSprites[spriteId], &sSubspriteTable_HofMonitorBig);
-    } else
+    }
+    else
```

Comment:
As above, if this style issue is in pret, it should be PR'd there

Reply:
Merged upcoming which includes the pret merge that fixes this